app/views/shared/metadata/edit/_study.html.erb

```diff
@@ -59,25 +59,22 @@
       %>
 
       <% metadata_fields.related_fields(to: :data_release_strategy, when: Study::DATA_RELEASE_STRATEGY_NOT_APPLICABLE) do %>
-        <%= group.select(:data_release_prevention_reason, Study::DATA_RELEASE_PREVENTION_REASONS) %>
-        <%= group.radio_select(:data_release_prevention_approval, Study::YES_OR_NO) %>
+        <%= group.select(:data_release_prevention_reason, study.data_release_prevention_options) %>
+        <%= group.text_area(:data_release_prevention_approval) %>
         <%= group.text_area(:data_release_prevention_reason_comment) %>
       <% end %>
       <% metadata_fields.related_fields(to: :data_release_strategy, in: Study::DATA_RELEASE_STRATEGIES, not: Study::DATA_RELEASE_STRATEGY_NOT_APPLICABLE) do %>
         <% metadata_fields.related_fields(to: :data_release_timing, when: Study::DATA_RELEASE_TIMING_DELAYED) do %>
-          <%= group.select(:data_release_delay_reason, Study::DATA_RELEASE_DELAY_REASONS_STANDARD) %>
+          <%= group.select(:data_release_delay_reason, study.data_release_delay_options) %>
           <%
             group.change_select_options_for(:data_release_delay_reason, when: :data_release_study_type_id, values: {
-              DataReleaseStudyType.assay_types.map(&:id)     => [ '' ] + Study::DATA_RELEASE_DELAY_REASONS_ASSAY,
-              DataReleaseStudyType.non_assay_types.map(&:id) => [ '' ] + Study::DATA_RELEASE_DELAY_REASONS_STANDARD
+              DataReleaseStudyType.assay_types.map(&:id)     => [ '' ] + study.data_release_delay_options(assay_option: true),
```

Comment:
Do we still need to keep the empty option?

Reply:
Hmm not sure. These empty options were already here, I guess its for a similar reason I added them elsewhere - to force a user selection.

Reply:
In that case, we can keep it I think.